index.d.ts

```diff
@@ -1902,21 +1906,33 @@ declare namespace Eris {
     permissionsOf(memberID: string | Member): Permission;
   }
 
+  export interface IntegrationApplication {
+    id: string;
+    name: string;
+    icon?: string;
```

Comment:
Nullable, not optional
```suggestion
    icon: string | null;
```